### ISSUE TITLE
did the sampler has distributed version?

### ISSUE THREAD

**Issue by 523997931** [KevinMusgrave/pytorch-metric-learning#546]
hello,
my train work used torch.utils.data.distributed.DistributedSampler to train on multi-GPU，now I'd like to use the per_class_sampler in my project, but I don't know is this sampler can used in distributed work directly?

**Comment by KevinMusgrave** [comment]
There's no distributed version. This wrapper might be useful: https://github.com/pytorch/pytorch/issues/23430#issuecomment-750037457

But I haven't tried it.
